handbook.md

```diff
@@ -0,0 +1,233 @@
+---
+layout: contribute
+title: Handbook
+---
+
+# How we make things
+
+## Branching and SCM
+
+We use git for source control. Our projects in
+[theforeman](https://github.com/theforeman) follow this model for branching:
```

Comment:
It's only the four core projects (foreman, smart-proxy, foreman-installer, foreman-selinux) that follow this exact model, most plugins don't - or pick and choose bits.